pymc4/gp/cov.py

```diff
@@ -66,28 +66,29 @@
         entry of the list, the leftmost ``n`` dimensions of ``i'th`` ``feature_ndims`` column
         are considered for evaluation.
     scale_diag : {Number, array_like}, optional
-        Scaling parameter of the lenght_scale parameter of stationary kernels for
-        performing Automatic Relevance Detection (ARD). Ignored if keyword argument ``ARD=False``."""
+        Scaling parameter of the lenght scales of stationary kernels. When not `None`,
```

Comment:
misspelled "length"

Reply:
Fixed. Thanks! And sorry for getting back late on this.